ci/authn-azure/policies/azure-hosts.yml

```diff
@@ -7,6 +7,13 @@
     - &hosts
       - !host
         id: test-app
+        annotations:
+          authn-azure/subscription-id: test-subscription
+          authn-azure/resource-group: test-group
+          authn-azure/system-assigned-identity: test-app-pipeline
+
+      - !host
+        id: user-app
         annotations:
           authn-azure/subscription-id: test-subscription
           authn-azure/resource-group: test-group
```

Comment:
please change to `user-assigned-identity-app`

Reply:
also please add an annotation for the user-assigned-identity-app

Reply:
Updating `user-assigned-identity-app`

The user-assigned-identity annotation appears in the actual file if you open it